zerver/migrations/0027_stream_mandatory_stream.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('zerver', '0026_delete_mituser'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='stream',
+            name='mandatory_stream',
```

Comment:
Might be better to name this `mandatory` instead of `mandatory_stream` (since it's already in the `stream` model so adding `_stream` at the end seems redundant)